docs/development.md

```diff
@@ -0,0 +1,107 @@
+# Development setup
```

Comment:
Might want to mention the FF SAML extension as well as the Chrome SAML extension, really nice when having to debug SAML related things.